spec/models/rss_feed_url_spec.rb

```diff
@@ -115,6 +115,20 @@
         end
       end
     end
+
+    context 'when the last_crawl_status is greater than 255 characters' do
+      before do
+        rss_feed_url.update(last_crawl_status: 'x' * 500)
+      end
+
+      it 'should truncate the value to 255 characters' do
+        expect(rss_feed_url.last_crawl_status.length).to eq(255)
+      end
+
+      it 'should be valid' do
```

Comment:
I think we can skip this test, as we are using truncation rather than a length validation for `last_crawl_status`.